tests/test_analytical.py

```diff
@@ -36,7 +36,7 @@ def test_convergence_to_analytical_solution():
     t_d = t_loss * t_p / (t_loss + t_p)
 
     analytical_profile = (
-        1 / np.sqrt(np.pi) * t_d / t_w * amp * np.exp(-x / (v_p * t_loss))
+        1 / np.sqrt(2 * np.pi) * t_d / t_w * amp * np.exp(-x / (v_p * t_loss))
```

Comment:
Doesn't this use exp pulse shape? How come we changed the analytical expression if the pulse shape was not changed?

Reply:
That's because the blobs still are Gaussian in y